src/main/java/com/google/sps/data/Event.java

```diff
@@ -11,14 +11,14 @@
 public final class Event{
 
     private String type;
-    private String details;
+    private String description;
     private double price;
-    private long epoch;
+    private String date;
```

Comment:
Although datastore has an epoch value, the `event` object converts it to date when the object is created. It made more sense  to use date.